### PR TITLE
43 fix getstarted png images

### LINKED ISSUE #22: Links to app catalog
Add the missing links in the callouts

              Link to the section directly

Originally posted by @nitindermohan in
-  https://github.com/nitindermohan/oak-new-docs/pull/7#discussion_r1865999784
- https://github.com/nitindermohan/oak-new-docs/pull/7#discussion_r1866266290
- https://github.com/nitindermohan/oak-new-docs/pull/7#discussion_r1866240477
- https://github.com/nitindermohan/oak-new-docs/pull/7#discussion_r1865989616

### LINKED ISSUE #23: fix unikernel page links
Fix the uniekrnel page links when everything is merged

> This is definitely requires a link, but it won't work until this branch is rebased onto the app-catalog branch

_Originally posted by @Mjaethers in https://github.com/nitindermohan/oak-new-docs/pull/16#discussion_r1871545957_

### LINKED ISSUE #43: Fix getstarted png images
Getstarted images are low res, better to swap them with svgs.

### PR DESCRIPTION
**Pull request by giobart**
fixes #43 
fixes #22 
fixes #23

### CODE REVIEW

**Comment by Malyuk-A** [comment]
Was it an intentional design decision from you end to add glow/drop shadows?

image

**Comment by Malyuk-A** [comment]
Is this very narrow design also intended?

image

**Comment by giobart** [comment]
> Was it an intentional design decision from you end to add glow/drop shadows?
> 
> image

I'm not 100% sure what you're referring to tbh, so maybe it was not intentional :D All I did compared to prev pic is remove the filling...

**Comment by giobart** [comment]
> Is this very narrow design also intended?
> 
> image

Ofc not, but where is this menu?

**Comment by Malyuk-A** [comment]
> > Was it an intentional design decision from you end to add glow/drop shadows?
> > image
> 
> I'm not 100% sure what you're referring to tbh, so maybe it was not intentional :D All I did compared to prev pic is remove the filling...

I am talking about the white backdrop:

here a comparison without it

image

I do not mind - just curious if this was intentional or not:
image

**Comment by Malyuk-A** [comment]
> > Is this very narrow design also intended?
> > image
> 
> Ofc not, but where is this menu?

OK NVM - this is the case for every single "folder":

e.g. Dashboard Features is a folder containing proper pages - Dashboard Features itself is not a real standalone page:
image

But in the search function of hugo you can search and open exactly that:

image

image

File: content/docs/manuals/app-catalog/nginx-server-unikraft.md [hunk 1]
```diff
@@ -109,14 +116,21 @@ In this guide we'll use the Oakestra CLI tool to interact with the Oakestra plat
 
  {{< callout context="tip" title="Oakestra CLI Tool" icon="outline/rocket">}}
 
-In this guide we'll use the comprehensive Oakestra CLI toolkit to interact with the Oakestra-managed infrastructure. To find out more about the CLI tool, please refer to the CLI section of the manuals.
+In this guide we'll use the comprehensive Oakestra CLI toolkit to interact with the Oakestra-managed infrastructure. 
 
+{{< link-card
```
Comment (Malyuk-A) [inline content/docs/manuals/app-catalog/nginx-server-unikraft.md:121]: good to know that you can in fact nest a link-card inside a callout - did not work for me when I tried it a few times 👀

**Comment by giobart** [comment]
This issue is specific for the list.html template of Doks, I created a new issue for this. Thanks for noticing it :D

**Review by Malyuk-A — APPROVED** [review]
Looks good to me - I have checked each changed link and figure

**Review by nitindermohan — APPROVED** [review]
Looks good to me. Approved

**Pull request merged**